org.tuxpaint.Tuxpaint.json

```
@@ -5,6 +5,7 @@
     "sdk": "org.freedesktop.Sdk",
     "finish-args": [
         "--device=dri",
+        "--filesystem=xdg-pictures",
```

I think this could be:

```suggestion
        "--filesystem=xdg-pictures/TuxPaint",
```

---

but I need to test it